components/script/dom/canvasrenderingcontext2d.rs

```diff
@@ -156,6 +143,58 @@ impl LayoutCanvasRenderingContext2DHelpers for LayoutDom<'_, CanvasRenderingCont
     }
 }
 
+#[cfg_attr(crown, allow(crown::unrooted_must_root))] // ID is not jsmanaged
```

Comment:
I'm not sure I understand what this comment refers to. Can you explain a bit what you mean here?

Reply:
I opened https://github.com/servo/servo/issues/35570 and linked it in code.